compose-common.yml

```diff
@@ -12,35 +12,49 @@ base:
     # --link some-rabbit:rabbit "just works"
     - DJANGO_CELERY_BROKER_URL=amqp://guest@rabbit
     - DJANGO_DATABASE_URL=postgis://postgres@database/postgres
-    - DJANGO_EMAIL_HOST=localhost
-    # Settings above likely need to be changed, depending on  the environment
+    - APP_PORT=8000
+    - APP_HOST=0.0.0.0
+
+    # Logging
+    - PYTHONUNBUFFERED=non-empty-string
+    - PYTHONIOENCODING=utf-8
+    - DJANGO_LOG_LEVEL=INFO
+    # https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/#python-options
+    - PYTHONHASHSEED=random
+
+
+    # Settings only need to be changed, if docker-compose is being changed
+    # or not used at all.
     ##########################################################################
```

Comment:
good catch, actually, no, should be 

the settings _above_ only need to be changed...